### DISCUSSION TITLE
pub-debugger code and blog teaser_marker issues

### DISCUSSION THREAD

**Discussion by AgedLace** [mkdocs-publisher/mkdocs-publisher#29] [General]
If one cuts and pastes the debugger code into their mkdocs.yml file, the indentations are off.  It threw a plugin config error.

As for the blog, the teaser_marker: should use the 'apostraphe' as opposed to the backticks.  

Will have to finish my testing over the weekend.  Will keep you posted.

**Comment by mkusz** [comment]
Thx for that findings. Will push with next release.
